scripts/cancelBatch.py

```diff
@@ -0,0 +1,53 @@
+# Script that takes in an array of batch names and applies take a bulk action such as removing the unique id or canceling all tasks.
+# By default, this script makes 50 concurrent API calls.
+import scaleapi
+from scaleapi.exceptions import ScaleException
+
+import argparse
+from concurrent.futures import ThreadPoolExecutor
+
+# Change this for update concurrency
+MAX_WORKERS = 50 
+
+# globals
+task_count = 0
+task_errors = []
+tasks_cancelled = []
+
+def main():
+
+    args = get_args()
+    clearUniqueId = args.clear or False
+
+    client = scaleapi.ScaleClient(args.api_key)
+    preProcessedBatches = args.batches.split(",")
+    batches = [word.strip() for word in preProcessedBatches]
+    for batch in batches:
+
+        task_count = client.get_tasks_count(batch_name = batch)
+        print("Batch ", batch, " contains ", task_count, " tasks.")
+        
+        tasks = client.get_tasks(batch_name = batch)
+        with ThreadPoolExecutor(max_workers=MAX_WORKERS) as executor:
+            for task in tasks:
+                try:
+                    tasks_cancelled.append(executor.submit(client.cancel_task(task.task_id, clearUniqueId)))
```

Comment:
will it work in that way and exception will work since this will be kinda in a thread pool. I doubt that (although never tested). I would suggest having cancel as another method, which will handle exception and either return success of failure, then handle counts afterwards. 

Example here: https://github.com/scaleapi/data-conversion/blob/4de32ef6615e9e009aad96042065d43fc5847f6e/scripts/microsoft/audio/interruptions/create_interruption_tasks.py#L93-L101

I checked the `/cancel` endpoint and both completed tasks and the tasks with work started returns BadRequest (ScaleInvalidRequest exception). 

I think in this method, we can check if status is completed then we may send it to API (but count # of completed), so only case with the ScaleInvalidRequest exception will be counted as 'work already started tasks'. 

Already canceled tasks are safe to re-send.